chain/chain/src/chain.rs

```diff
@@ -3309,39 +3389,53 @@ pub fn check_refcount_map(chain: &mut Chain) -> Result<(), Error> {
             {
                 *block_refcounts.entry(prev_hash).or_insert(0) += 1;
             }
+            if !block_refcounts.contains_key(block_hash) {
+                block_refcounts.insert(*block_hash, 0);
```

Comment:
```rust
block_refcounts.entry(prev_hash).or_insert(0);
```

Reply:
I have no idea this is better, but ++ anyway.